OpenRA.Mods.Common/Traits/Capturable.cs

```diff
@@ -33,17 +34,11 @@ public bool CanBeTargetedBy(Actor captor, Player owner)
 			if (c == null)
 				return false;
 
-			var playerRelationship = owner.Stances[captor.Owner];
-			if (playerRelationship == Stance.Ally && !AllowAllies)
-				return false;
-
-			if (playerRelationship == Stance.Enemy && !AllowEnemies)
-				return false;
-
-			if (playerRelationship == Stance.Neutral && !AllowNeutral)
+			if (!c.CaptureTypes.Contains(Type))
 				return false;
 
-			if (!c.CaptureTypes.Contains(Type))
+			var stances = ForceTargetStances | TargetStances;
```

Comment:
This worries me a bit - in teamgame, this allow a player to capture a freshly-captured-by-his-ally actor just by arriving late.